tests/auth-sso-provisioning.unit.test.ts

```
@@ -1,29 +1,9 @@
-}
+import { AsyncLocalStorage } from 'node:async_hooks'
+import { describe, expect, it, vi } from 'vitest'
 
-const organizationStore = {
-  organizations: [] as Array<{ gid: string, role: string }>,
-  hasOrganizations: false,
-  fetchOrganizations: vi.fn(async () => {}),
-  dedupFetchOrganizations: vi.fn(async () => {}),
+interface MockFetchResponse {
+  ok: boolean
+  json(): Promise<Record<string, unknown>>
```

ESLint

[error] Shorthand method signature is forbidden. Use a function property instead.

(ts/method-signature-style)